lib/lorekeeper/json_logger.rb

```diff
@@ -96,7 +96,16 @@ def inspect
     end
 
     def write(message)
-      super(Oj.dump(message, mode: :compat, cache_keys: true, cache_str: 5) << "\n")
+      json_message =
+        begin
+          Oj.dump(message, mode: :compat, cache_keys: true, cache_str: 5)
+        rescue JSON::GeneratorError
+          Oj.dump(message)
+        rescue => e
+          Oj.dump(e.message)
```

Comment:
`e.message` should be a string so probably we should do something like
```suggestion
          Oj.dump(MESSAGE => e.message)
```

@jcarres-mdsol should we add `exception` and `backtrace` here or is it too much?
https://github.com/JordiPolo/lorekeeper/blob/20bb011bbd3fe08f8599da5c17e64130be1fde08/lib/lorekeeper/json_logger.rb#L79-L82

Reply:
@ykitamura-mdsol 

curious why suggested `MESSAGE` key is in all caps? 👀

Reply:
there is `MESSAGE` constant a few lines below 👇 

https://github.com/JordiPolo/lorekeeper/blob/d9f63ecb9cfbb098ff9219a378b783a84f3ceee8/lib/lorekeeper/json_logger.rb#L132

Reply:
thank you! understood